app/views/admin/split_details/edit.html.erb

```diff
@@ -7,15 +7,15 @@
 
 <%= simple_form_for @split_detail, url: admin_split_details_path, method: :put do |f| %>
   <%= f.error :base %>
-  <%= f.input :owner, label: "Owner" %>
-  <%= f.input :platform, collection: Split.platforms.keys %>
-  <%= f.input :location, label: "Location", hint: "Example: Sign Up" %>
+  <%= f.input :owner, label: "Owner", required: @split_detail.owner.present? %>
```

Comment:
Nice -- now that the model validations are conditional, I'm wondering if these `required:` options ever actually needed to be set, or if the Rails validations were driving the behavior all along.